bin/tests/common.rs

```diff
@@ -234,6 +239,9 @@ where
     let mut lines_buffer = String::new();
     for _safeguard in 0..100_000 {
         reader.read_line(&mut line).unwrap();
+        if line.is_empty() {
```

Comment:
Is this where the empty lines in output was coming from?

Reply:
yeah, when the process has exited, reading lines is a noop so we were writing empty lines to debug output.